tests/forms_tests/tests/test_media.py

```diff
@@ -646,3 +648,113 @@ def test_add_empty(self):
         merged = media + empty_media
         self.assertEqual(merged._css_lists, [{'screen': ['a.css']}])
         self.assertEqual(merged._js_lists, [['a']])
+
+
+@html_safe
+class Asset:
+    def __init__(self, path):
+        self.path = path
+
+    def __eq__(self, other):
+        return (
+            (self.__class__ == other.__class__ and self.path == other.path) or
+            (other.__class__ == str and self.path == other)
+        )
```

Comment:
Is this only required for the specifics of the test cases, or is it required to satisfy the merging and sorting in `Media` itself? If the former, all well and good, if the latter, would that need to be part of the `Paths as objects` interface contract?